imagestreams/postgresql-rhel.json

```diff
@@ -10,104 +10,90 @@
           "openshift.io/provider-display-name": "Red Hat, Inc.",
-          "description": "Provides a PostgreSQL 15 database on RHEL 9. For more information about using this database image, including OpenShift considerations, see https://github.com/sclorg/postgresql-container/blob/master/README.md.",
+          "description": "Provides a PostgreSQL 12 database on RHEL 7. For more information about using this database image, including OpenShift considerations, see https://github.com/sclorg/postgresql-container/blob/master/README.md.\n",
           "iconClass": "icon-postgresql",
           "tags": "database,postgresql",
-          "version": "15"
+          "version": "12"
         },
         "from": {
           "kind": "DockerImage",
-          "name": "registry.redhat.io/rhel9/postgresql-15:latest"
+          "name": "registry.redhat.io/rhscl/postgresql-12-rhel7:latest"
         },
         "referencePolicy": {
           "type": "Local"
         }
       },
       {
-        "name": "16-el8",
+        "name": "13-el7",
         "annotations": {
-          "openshift.io/display-name": "PostgreSQL 16 (RHEL 8)",
+          "openshift.io/display-name": "PostgreSQL 13 (RHEL 7)",
           "openshift.io/provider-display-name": "Red Hat, Inc.",
-          "description": "Provides a PostgreSQL 16 database on RHEL 8. For more information about using this database image, including OpenShift considerations, see https://github.com/sclorg/postgresql-container/blob/master/README.md.",
+          "description": "Provides a PostgreSQL 13 database on RHEL 7. For more information about using this database image, including OpenShift considerations, see https://github.com/sclorg/postgresql-container/blob/master/README.md.\n",
           "iconClass": "icon-postgresql",
           "tags": "database,postgresql",
-          "version": "16"
+          "version": "13"
         },
         "from": {
           "kind": "DockerImage",
-          "name": "registry.redhat.io/rhel8/postgresql-16:latest"
+          "name": "registry.redhat.io/rhscl/postgresql-13-rhel7:latest"
         },
         "referencePolicy": {
           "type": "Local"
         }
       },
       {
-        "name": "15-el8",
+        "name": "10-el8",
         "annotations": {
-          "openshift.io/display-name": "PostgreSQL 15 (RHEL 8)",
+          "openshift.io/display-name": "PostgreSQL 10 (RHEL 8)",
           "openshift.io/provider-display-name": "Red Hat, Inc.",
-          "description": "Provides a PostgreSQL 15 database on RHEL 8. For more information about using this database image, including OpenShift considerations, see https://github.com/sclorg/postgresql-container/blob/master/README.md.",
+          "description": "Provides a PostgreSQL 10 database on RHEL 8. For more information about using this database image, including OpenShift considerations, see https://github.com/sclorg/postgresql-container/blob/master/README.md.\n",
```

Comment:
The new line at the end is not needed.

Reply:
Good catch. That is some funny behaviour of yaml. I've fixed it with https://github.com/sclorg/postgresql-container/pull/549/commits/196423e6d8a834229b07d5e9c737104d925c5e70.